public/js/wp-agora-io-public.js

```diff
@@ -406,7 +440,8 @@ window.AGORA_UTILS = {
   },
 
   // REMOTE STREAMS UI
-  addRemoteStreamView: function(remoteStream) {
+  addRemoteStreamView: function(remoteStream, cond='') {
+    if(!window.isPublished && window.agoraMode!="audience") return; 
```

Comment:
what is the purpose of this `if` statement?

Reply:
> what is the purpose of this `if` statement?

Hi Hermes,

As in pre-call device test, remote stream will be added by stream-added event (even when local stream is not published), so to play the remote stream only after local stream is published(After the pre-call device test is finished), if(!window.isPublished), condition is added. And to play the same after pre call device is completed, called the same function addRemoteStream when stream is published.

Reply:
> what is the purpose of this `if` statement?

Fixed. Kindly check it.